docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,352 @@ This section only applies if a fixed seed is used:
   * Parameter order is irrelevant
   * Parameter names are case sensitive, PARAM:MY_PARAM != PARAM:myParam
 
-NORMAL
-,,,,,,
+NORMAL: Gaussian Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To set a normal (Gaussian) prior, use the keyword NORMAL. It takes two
-arguments, a mean value and a standard deviation. Thus, the following example
-will assign a normal prior with mean 0 and standard deviation 1 to the variable
-VAR1:
+The ``NORMAL`` keyword allows assigning a Gaussian (or normal) prior to a variable.
+It requires two arguments: a mean value and a standard deviation.
 
+Syntax
+~~~~~~
 ::
 
-   VAR1   NORMAL    0   1
+  VAR NORMAL <mean_value> <standard_deviation>
+
+Parameters
+~~~~~~~~~~
+- **<mean_value>**: Specifies the mean of the normal distribution.
+- **<standard_deviation>**: Indicates the standard deviation of the normal distribution.
```

Comment:
Probably re-use "Specifies" from above instead of "Indicates".
Or just start with "The" skipping entirely.

Reply:
I have updated all (I hope) definitions to just start with "The" as you suggest.